import_export_extensions/resources.py

```diff
@@ -141,8 +195,39 @@ def import_row(
                 else TaskState.PARSING.name
             ),
         )
+        if not force_import:
+            return imported_row
+        if (
+            imported_row.import_type == results.RowResult.IMPORT_TYPE_ERROR
+            or imported_row.import_type == results.RowResult.IMPORT_TYPE_INVALID
+        ):
+            imported_row.diff = []
+            for field in self.get_fields():
+                imported_row.diff.append(row.get(field.column_name, ""))
+
+            imported_row.non_field_skipped_errors.extend(
+                imported_row.errors,
+            )
+            if imported_row.validation_error is not None:
+                imported_row.field_skipped_errors.update(
+                    **imported_row.validation_error.error_dict,
+                )
+            imported_row.errors = []
+            imported_row.validation_error = None
+
+            imported_row.import_type = results.RowResult.IMPORT_TYPE_SKIP
```

Comment:
I think it would be nice to make it shorted

1. Move `skip` logic into separate method like `_skip_row_with_errors`
2. Simplify `if` statement like:

```python
...

if force_import and imported_row.has_error_import_type:
	imported_row = _skip_row_with_errors(imported_row)

return imported_row
```